.github/workflows/e2e.yaml

```diff
@@ -26,6 +26,10 @@ jobs:
     - name: Install drenv
       run: pip install -e test
 
+    - name: Setup drenv
+      working-directory: test
+      run: drenv setup envs/regional-dr.yaml
+
```

Comment:
I see the CI is green, which means this was exercised, and 1.33 works?

Reply:
No, since we do not install yet the tool dynamically in the CI, this just shows that the changes are compatible with 1.32.0 (version installed in the CI runner).